src/DataTool.php

```diff
@@ -173,6 +173,29 @@ public function setFields($fieldDefs)
                 continue;
             }
 
+            // enum probability option
+            if (!empty($fieldRules['enum_key_probabilities'])
+                && !empty($GLOBALS['app_list_strings'][$fieldDef['options']])) {
```

Comment:
Please remove this check. If the app list strings are not defined for this enum field then it should generate a lot of notices instead of skipping the field silently.

If the reason that you've added this check is that there is some enum field whose app list strings are not defined, then please let me know.

Reply:
This check not for `app_list_strings` only, but for `options` too.
Opportunity - commit_stage has empty `options` like a plenty of others. This check was here before and without it I catch lots of notices.